tests/ui/consts/issue-25826.stderr

```diff
@@ -1,11 +1,11 @@
-error[E0277]: can't compare `*const ()` with `*const ()` in const contexts
+error[E0277]: can't compare `*const ()` with `_` in const contexts
   --> $DIR/issue-25826.rs:3:52
    |
 LL |     const A: bool = unsafe { id::<u8> as *const () < id::<u16> as *const () };
-   |                                                    ^ no implementation for `*const () < *const ()` and `*const () > *const ()`
+   |                                                    ^ no implementation for `*const () < _` and `*const () > _`
    |
-   = help: the trait `~const PartialOrd` is not implemented for `*const ()`
-note: the trait `PartialOrd` is implemented for `*const ()`, but that implementation is not `const`
+   = help: the trait `~const PartialOrd<_>` is not implemented for `*const ()`
+note: the trait `PartialOrd<_>` is implemented for `*const ()`, but that implementation is not `const`
```

Comment:
Those diagnostics changes are kind of bad, but I'm not really sure what causes them / how to fix them